src/Bridge/Symfony/Routing/Router.php

```diff
@@ -30,11 +30,20 @@
         UrlGeneratorInterface::NET_PATH => RouterInterface::NETWORK_PATH,
     ];
 
+    const CONFIG_MAP = [
+        'ABS_URL' => UrlGeneratorInterface::ABS_URL,
+        'ABS_PATH' => UrlGeneratorInterface::ABS_PATH,
+        'REL_PATH' => UrlGeneratorInterface::REL_PATH,
+        'NET_PATH' => UrlGeneratorInterface::NET_PATH,
+    ];
+
     private $router;
+    private $defaultReferenceType;
 
-    public function __construct(RouterInterface $router)
+    public function __construct(RouterInterface $router, string $defaultReferenceType)
```

Comment:
I seem to be missing a point..

I imagine an implementation like this:

```php
    public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH) : string
    {
        /* [...] */
        return $this->router->generate($routeName, ['id' => implode(';', $identifiers)], $referenceType ?? $this->defaultReferenceType);
    }
```

But if we set $referenceType in the method signature to UrlGeneratorInterface::ABS_PATH, it will always be set and $this->defaultReferenceType will never be used.